### ISSUE TITLE
Literal String Interpolation autoquotation not respondig to config.

### ISSUE THREAD

**Issue by Carbaz** [Microsoft/vscode#51346]
with`"editor.autoClosingBrackets": false` literal string interpolation lines are still automatically writing the closing quotes.
```
f''
f""
```
Same happens with Unicode strings, byte strings, raw strings...

**Comment by rebornix** [comment]
You can set `editor.autoClosingQuotes` to `false` to disable this feature.

**Comment by Carbaz** [comment]
No, it doesn't work.

First, you mean `editor.autoClosingQuotes` setted to **`never`**, it changed since I opened the issue.

And it also does not work _and even if it had worked it would had been losing the capability to use "languageDefined" option, so it would be only a workaround over an still existing issue_.

On **Python** files, if you write `f'` , even with the settings, both, setted to "never" it auto closes the string and you get `f''`

The problem is with the Python prefixed strings specifically.

To be more clear, this lines has been written with only one keypress for quotes and  `editor.autoClosingQuotes` setted to `false`:

```
f""
u""
b""
a"
r""
f''
u''
b''
a'
r''
```

This only happen if the language is setted to **Python** and seems to be related to prefixed python strings treatment.

So for the exactly same key presses: `[", F, o, u, n, d,  , v, a, l, u, e, s, :,  , {, l, i, s, t, (, g, v, a, l, ), }, "]`
 you get different outputs for prefixed and non prefixed strings:

```
"Found values: {list(gval)}"
f"Found values: {list(gval)}""
```

P.D. Tested with all my Python extensions disabled.
